api/types.go

```diff
@@ -38,6 +39,19 @@ type networkCreate struct {
 	Options     map[string]interface{} `json:"options"`
 }
 
+// sandboxCreate is the expected body of the "create sandbox" http request message
+type sandboxCreate struct {
+	ID                string                 `json:"id"`
```

Comment:
Does the sandbox ID carry container-id and is expected the caller must call with an unique container-id / sandbox-id ?